aroma/plotting.py

```diff
@@ -53,8 +53,8 @@ def classification_plot(myinput, out_dir):
     # get counts
     ncomp = len(df)
     nmot = len(df.loc[df[0] == "True"])
-    LGR.info('Found', nmot, 'head motion-related components in a total of',
-             ncomp, 'components.')
+    LGR.info(f'Found {nmot} head motion-related components in a total of',
```

Comment:
Needs a space at the end.

Reply:
Also, f-strings limit us to 3.6+, I believe. I never planned to support Python 2, but I haven't thought about 3.5.

Reply:
Ah, it looks like we have `setup.cfg` limited to 3.6+, so it's all good. We just need the space at the end of the first line.